cli/CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 13.12.0
+
+_Released 6/18/2024 (PENDING)_
+
+**Features:**
+
+- Adds Component Testing support for Angular version 18. Addresses [#29309](https://github.com/cypress-io/cypress/issues/29309).
+
+**Bugfixes:**
+
+- Fixed an issue where `inlineSourceMaps` was still being used when `sourceMaps` was provided in a users typescript config for typescript version 5. Fixes [#26203](https://github.com/cypress-io/cypress/issues/26203).
```

Comment:
This is a wonderful fix BTW. I run into this all the time when running Angular apps with CT 👍

Reply:
Thank you. It actually went in with #29568 but I forgot to update the changelog after release so the change entry was in the wrong cypress version 😬 .